### ISSUE TITLE
Feature Request: I2P Integration

### ISSUE THREAD

**Issue by SamsungGalaxyPlayer** [m2049r/xmrwallet#108]
Similar to #100 

It may make sense to wait for Kovri, but there is an existing [I2P application](https://play.google.com/store/apps/details?id=net.i2p.android) for android. This feature would be helpful for people who wish to connect to an I2P remote node, such as those on [MoneroWorld](https://moneroworld.com/#nodes).

The android app linked uses an HTTP/HTTPS proxy on port 4444.

**Comment by Engelberg** [comment]
It's becoming easier and easier to set up a personal i2p monero node, with a guide on the official monero site: https://www.getmonero.org/resources/user-guides/node-i2p-zero.html

It would be ideal to be able to connect one's mobile wallet to one's personal i2p node.

**Comment by jbakosi** [comment]
Indeed, this would be nice. I do not believe there is any other android monero wallet that would support i2p, except for the official cli, via termux. Also, monerod nicely runs with [purple i2pd](https://github.com/PurpleI2P/i2pd) (a c++ implementation of i2p) and there is also a corresponding [purple i2pd android app](https://github.com/PurpleI2P/i2pd-android) as well which can be configured to create a tunnel.
